themes/bootstrap3/js/facets.js

```diff
@@ -1,5 +1,81 @@
 /*global VuFind */
-/*exported initFacetTree */
+/*exported initFacetTree, setupFacetList */
```

Comment:
Might it be a better idea to register setupFacetList in the VuFind object instead of exporting another global function? I realize there's still a lot of legacy code that does things this way, but we're trying to stay away from adding new global functions if it's possible to do things inside the VuFind namespace instead.

Reply:
The new JS functions have been encapsulated into a `facetList` submodule.